README.md

```diff
@@ -1,4 +1,4 @@
-Seltzer CRM 0.7.9 - An open source CRM for hackerspaces  
+Seltzer CRM 0.7.11 - An open source CRM for hackerspaces  
```

Comment:
Straight from 9 to 11?

Reply:
the form update in 836650edd98e14fee06edf285d1060f9a88501d9 was .10 :)